elasticdl/python/master/master.py

```diff
@@ -127,6 +147,18 @@ def __init__(self, args):
         self._should_stop = False
         self._exit_code = 0
 
+    def _set_completed_steps_by_checkpoint(self, checkpoint_dir_for_init):
+        if not checkpoint_dir_for_init:
+            return
+
+        CheckpointSaver.check_checkpoint_valid(checkpoint_dir_for_init)
```

Comment:
If not valid, return

Reply:
Yes, I have added the code to raise a `ValueError` if invalid.